dashboard/test/models/script_test.rb

```diff
@@ -681,29 +681,29 @@ def populate_cache_and_disconnect_db
   end
 
   test 'should summarize script' do
-    script = create(:script, name: 'single-stage-script')
-    stage = create(:stage, script: script, name: 'Stage 1')
-    create(:script_level, script: script, stage: stage)
+    script = create(:script, name: 'single-lesson-script')
+    lesson = create(:lesson, script: script, name: 'lesson 1')
+    create(:script_level, script: script, lesson: lesson)
     script.teacher_resources = [['curriculum', '/link/to/curriculum']]
 
     summary = script.summarize
 
     assert_equal 1, summary[:stages].count
-    assert_nil summary[:peerReviewStage]
+    assert_nil summary[:peerReviewlesson]
```

Comment:
peerReviewLesson (upper case L)?

Reply:
Also was this updated to this name because searching for it did not find anything and I see another one down below that is still peerReviewStage

Reply:
Good catch, that probably would've broken the test's ability to detect problems... I'll change it back to peerReviewStage.